### ISSUE TITLE
Update Django to 1.11.0 (LTS)

### ISSUE THREAD

**Issue by alanmoo** [mozilla/foundation.mozilla.org#650]
@cadecairos commented on [Wed Apr 19 2017](https://github.com/mozilla/network-api/issues/130)

https://www.djangoproject.com/weblog/2017/apr/04/django-111-released/

Lets upgrade Django to 1.11.0, which is an LTS release and will recieve update untill December 2017.

---

@cadecairos commented on [Wed Apr 19 2017](https://github.com/mozilla/network-api/issues/130#issuecomment-295409136)

cc @gideonthomas @Pomax @simonwex 

---

@gideonthomas commented on [Thu Apr 20 2017](https://github.com/mozilla/network-api/issues/130#issuecomment-295905611)

sounds good! There doesn't seem to be any changes that would break stuff we wrote but don't know about what would break with Mezzanine.

---

@cadecairos commented on [Mon May 08 2017](https://github.com/mozilla/network-api/issues/130#issuecomment-299948407)

Blocked on https://github.com/stephenmcd/mezzanine/issues/1749

**Comment by Pomax** [comment]
relevant link: https://docs.djangoproject.com/en/1.11/releases/1.11/

**Comment by Pomax** [comment]
already done \o/
